src/coreclr/vm/jitinterface.cpp

```diff
@@ -1170,6 +1170,16 @@ void CEEInfo::resolveToken(/* IN, OUT */ CORINFO_RESOLVED_TOKEN * pResolvedToken
             th = ClassLoader::LoadArrayTypeThrowing(th);
             break;
 
+        case CORINFO_TOKENKIND_Casting:
+            // Disallow ELEMENT_TYPE_BYREF and ELEMENT_TYPE_VOID
```

Comment:
This is duplicated for 3rd time here. We should move it in front of the switch under `if (tokenType != CORINFO_TOKENKIND_Ldtoken)`